src/panels/developer-tools/state/developer-tools-state.js

```diff
@@ -157,7 +159,20 @@ class HaPanelDevState extends EventsMixin(LocalizeMixin(PolymerElement)) {
                 title="[[localize('ui.panel.developer-tools.tabs.states.more_info')]]"
               >
               </paper-icon-button>
-              <a href="#" on-click="entitySelected">[[entity.entity_id]]</a>
+              <paper-icon-button
+                id="copyBtn-[[sanitizeId(entity.entity_id)]]"
+                on-click="entityCopyId"
+                icon="hass:content-copy"
+                alt="[[localize('ui.panel.developer-tools.tabs.states.copy_entity_id')]]"
+                title="[[localize('ui.panel.developer-tools.tabs.states.copy_entity_id')]]"
+              >
+              </paper-icon-button>
+              <a
+                href="#"
+                on-click="entitySelected"
+                id="entity-id-[[sanitizeId(entity.entity_id)]]"
```

Comment:
What is the use of this?

Reply:
Removed (see [previous comment](https://github.com/home-assistant/home-assistant-polymer/pull/4259#discussion_r349073722))